template_app/package.json

```diff
@@ -10,11 +10,12 @@
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
     "lint": "prettier --check . && eslint .",
-    "format": "prettier --write .",
+    "format": "prettier --write src tests *.json *.ts *.js",
     "test:integration": "playwright test",
     "test:unit": "vitest",
     "wasm": "wasm-pack build --dev --target web ../rust_backend",
-    "wasm-release": "wasm-pack build --release --target web ../rust_backend"
+    "wasm-release": "wasm-pack build --release --target web ../rust_backend",
+    "setup-pyodide": "curl -L https://github.com/pyodide/pyodide/releases/download/0.26.0/pyodide-0.26.0.tar.bz2 | tar -xjf - -C static"
```

Comment:
This is expensive. I couldn't figure out how to get a minimal pyodide setup working that includes micropip, but not all of the prebuilt wheels